react-cardknox-ifields/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cardknox/react-ifields",
-  "version": "2.2.60",
+  "version": "2.2.60-beta.4",
```

Comment:
Should this be committed?

Reply:
yes

Reply:
@mstein-ck this is the version that is currently out

Reply:
I think we should keep the version at the proposed stable version when this release goes live.
We could a better way of publishing that can dynamically update the version but until then I don't think every beta version number should need to be committed to master
Let me know if you disagree

Reply:
It's not getting published automatically, so committing to master will not put this release live.
When we're ready to make this release live we'll update mater with appropriate version.
I believe master should reflect the latest published version.